### PR TITLE
feat: Feature Operator

### PR DESCRIPTION
**Pull request by vinnyjth**
This PR mimics the `ContentItem.getFeatures` code from Core. I didn't port in the whole method, since some of it we were planning on deprecating :)

### CODE REVIEW

**Comment by vinnyjth** [comment]
Screen Shot 2021-07-07 at 12 19 54 PM

File: dags/rock-content-item-backfill-core-dag.py [hunk 1]
```diff
@@ -28,53 +29,59 @@
          # catchup=False # enable if you don't want historical dag runs to run
          ) as dag:
 
-    t0 = PythonOperator(
+    base_items = PythonOperator(
```
Comment (vinnyjth) [inline dags/rock-content-item-backfill-core-dag.py:32]: I renamed the operators with actually helpful names, they were getting hard to manage :)

File: dags/rock_features.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        pg_hook.insert_rows(
+            '"features"',
+            list(insert_features),
+            columns,
+            0,
+            True,
+            replace_index=('"parentId"', '"type"', '"data"')
+        )
+
+        add_apollos_ids = """
```
Comment (tomthornton) [inline dags/rock_features.py:189]: It appears that "originType" is not a column on the features table. Are there more recent migrations that adds this?

**Review by tomthornton — CHANGES_REQUESTED** [review]
Getting all of the features imported but the apollos ID isn't updating because there doesn't seem to be an `originType` column on the features table

File: dags/rock_features.py (hunk 2 above)
Comment (vinnyjth) [reply dags/rock_features.py:189]: Good catch! That's a miss on my part.

**Comment by vinnyjth** [comment]
@tomthornton Updated to accommodate that this table doesn't have/need an originId.

**Review by tomthornton — APPROVED** [review]
Great looks good!

**Pull request merged**